packages/sftp-client/src/index.ts

```diff
@@ -41,6 +41,13 @@ export class SftpClient {
     this.clientOptions = {
       host: config.host,
       port: get(config, 'port', 22),
+      algorithms: {
+        kex: {
+          append: ['diffie-hellman-group14-sha1'],
```

Comment:
Is there a reason not to append the rest of the optional algorithms (potential removal causing an error to be thrown, etc)?

Reply:
Nope! I originally scoped it to just the one that's called out in the ticket but that's not a strong opinion. I'll add the rest 👍

Reply:
Rather be more restrictive than overly accepting I agree; there should be a reason `diffie-hellman-group14-sha1` was not included by default.